docs/content/test/ReportViewer.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Report Viewer
+type: docs
+---
+
+# Report Viewer (internal)
+
+Используем отчеты в [Report Viewer (internal)](http://links.k.avito.ru/cfxRp7KAg), которые содержат:
+
+- **Видео** с эмулятора
```

Comment:
It's not clear where i can get these artifacts

Reply:
I've just moved the text.
All of these features is obvious when you see the report. Agree, that it's better to make it more tutorial way, but not in this PR.